### DISCUSSION TITLE
Cypress intercept throwing "jwt malformed" error on unauthorized queries

### DISCUSSION THREAD

**Discussion by MohsinGhani** [cypress-io/cypress#29031] [Questions and Help]
### Issue Description
When using Cypress intercept to handle unauthorized queries, an error "jwt malformed" is thrown. This issue arises specifically when intercepting unauthorized queries. It's worth noting that authorized queries, after logging in, function as expected without throwing errors. Additionally, manual testing has confirmed that the intercept functionality works as intended.

### Steps to Reproduce
1. Log in with one user account.
2. Perform actions that trigger queries.
3. Log out.
4. Attempt to perform actions that trigger queries again.

### Expected Behavior
Unauthorized queries intercepted after logging out should not result in a "jwt malformed" error. Instead, they should be handled gracefully as per the intercept logic.

### Actual Behavior
After logging out, all queries intercepted result in a "jwt malformed" error.

**Environment**
Cypress Version: 13.5.1
Operating System: Windows 10
Additional relevant environment details: Graphql server on nest js

### Workarounds
As a workaround, manually testing the functionality confirms that it works as expected. However, this is not a sustainable solution for automated testing scenarios.

**Comment by kirillvladimirovwork** [comment]
Hey, were you able to find a solution?
